style.css

```diff
@@ -621,3 +621,334 @@ hr {
   position: absolute;
   bottom: 1rem;
 }
+
+@media (min-width: 769px) and (max-width: 1920px) {
+  body {
+    margin: auto;
+    min-width: 769px;
+    max-width: 1920px;
+    overflow-x: hidden;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .body {
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #fff;
+    height: 75px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    width: 100%;
+  }
+
+  nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+
+  header .logo {
+    font-weight: 700;
+    font-size: 18px;
+    color: #6070ff;
+    margin-left: 5rem;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    text-align: center;
+    margin-right: 5rem;
+  }
+
+  nav ul li a {
+    text-decoration: none;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    font-family: 'Poppins', sans-serif;
+    margin-left: 1.5rem;
+  }
+
+  .showcase {
+    background-image: url(img/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    transform: translateY(10.8%);
+    padding: 1rem;
+  }
+
+  .showcase .title {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: start;
+    margin: 1rem 15%;
+    padding-top: -5rem;
+  }
+
+  .title h1 {
+    font-weight: 700;
+    font-size: 48px;
+    color: #091e42;
+  }
+
+  .title p {
+    margin-bottom: 2rem;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    height: 122px;
+  }
+
+  .title h2 {
+    font-weight: 500;
+    font-size: 16px;
+    color: #7f8cff;
+    margin-top: -0.5rem;
+  }
+
+  .work {
+    padding: 15% 7%;
+    padding-bottom: 10%;
+  }
+
+  .card-container {
+    grid-template-columns: repeat(2, 1fr);
+    column-gap: 5rem;
+    row-gap: 1rem;
+    padding: 4%;
+    width: 100%;
+    box-shadow: 5px 10px 20px 12px rgba(0, 0, 0, 0.1);
+  }
```

Comment:
You've done an excellent job with the styling of the cards on desktop! I wanted to offer **a few friendly suggestions to make them even closer to the [provided templat](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122)e:**

- [ ] Would you mind checking the `padding` of your cards? It seems **slightly larger than the template.** 😉
- [ ] Additionally, could you please review the alignment of the text within the cards? It appears centered, but according to the template, **it should be positioned slightly higher.** 🙋‍♂️

   ### Your design:
   ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/107952724/f7460cd4-a463-4509-8bbd-6981e3433337)

   ### Figma design:
   ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/107952724/fb9106ee-2929-4d83-986c-9ff74a6c81bb)